source/mir/random/flex/internal/area.d

```diff
@@ -186,13 +190,13 @@ body
         }
         else if (iv.c == 1)
         {
-            area = S(0.5) * sh.a * intLength * (z + 2);
+            area = S(0.5) * sh.a * ivLength * (z + 2);
```

Comment:
>  Do you mean beta = g'(x0) ... ?

Yes as said, it's pre-draft. Thanks for spotting :)

> n the same document, when you write g(x) do you mean what in the Tinflex paper is given as Tc(x) ... ?

Yes `g(x)` was `\tilde{f}(x)`. Both functions are the result of the transformation. I updated the "Supplementary explanation doc" with a hopefully understandable explanation.
Don't hesitate to point out further problem in there.